ajax/libs/select-or-die/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "select-or-die",
+  "filename": "selectordie.min.js",
+  "version": "0.1.8",
+  "author": "Per Vestman",
+  "license": "MIT",
+  "dependencies": {
+    "jquery": ">=1.8"
+  },
+  "description": "Yet another jQuery plugin to style/replace select elements.",
+  "homepage": "http://sod.all-zippy.com/",
```

Comment:
Because we cannot access http://vst.mn/selectordie. That's the url someone mirrored. https://github.com/vestman/Select-or-Die/issues/54
Or it's better to remove homepage field?